NetExercise/Array/Exercise06Array.cs

```diff
@@ -0,0 +1,30 @@
+namespace NetExercise.Array
+{
+    public class Exercise06Array
+    {
+        public static int BinarySearch(int[] arr,int a)
+        {
+            var l = 0;
+            var r = arr.Length - 1;
```

Comment:
Implement chưa thực sự chuẩn lắm

Em gợi ý sửa 2 dòng này như thế này, anh sửa nốt những chổ còn lại cho đúng nha

```suggestion
            var l = -1;
            var r = arr.Length;
```

Reply:
done